src/modules/rest_countries.js

```diff
@@ -0,0 +1,52 @@
+class Countries {
+  constructor() {
+    this.selectedCountries = [
+      'turkey',
+      'russia',
+      'canada',
+      'iran',
+      'egypt',
+      'colombia',
+      'afghanistan',
+      'japan',
+      'china',
+      'india',
+      'iraq',
+      'nigeria',
+      'france',
+      'italy',
+      'spain',
+      'thailand',
+      'australia',
+      'brazil',
+      'argentina',
+      'pakistan',
+      'mexico',
+      'venezuela',
+      'UK',
+      'USA',
```

Comment:
The selected countries are assertive and according to the project's requirements. Good job with this!